pkg/maps/ipcache/ipcache.go

```diff
@@ -248,3 +252,28 @@ var (
 func Reopen() error {
 	return IPCache.Map.Reopen()
 }
+
+// Function to update IPCache map with node PodCIDR, VTEP CIDR
+func UpdateIPCacheMapping(newCIDR *cidr.CIDR, newTunnelEndpoint net.IP,
+	securityIdentity uint32, vtepMAC mac.MAC, encryptKey uint8) error {
+
+	key := NewKey(newCIDR.IP, newCIDR.Mask)
+
+	mac, err := vtepMAC.Uint64()
+	if err != nil {
+		log.Warningf("invalid MAC: %v", err)
+		return err
```

Comment:
There is no need to warn and return the error. If this code is returning the error, then presumably the caller should sufficiently log it.
```suggestion
		return err
```

Reply:
done